From a8cc033029b9f0a2243eb0dbf41a52deeed6b79d Mon Sep 17 00:00:00 2001
From: Christoph Junghans <junghans@lanl.gov>
Date: Wed, 10 May 2017 13:52:45 -0600
Subject: [PATCH] cmake: link mpi_interop against mpi

---
 examples/mpi_interop/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/examples/mpi_interop/CMakeLists.txt b/examples/mpi_interop/CMakeLists.txt
index c76a21c..dd2f58c 100644
--- a/examples/mpi_interop/CMakeLists.txt
+++ b/examples/mpi_interop/CMakeLists.txt
@@ -22,8 +22,11 @@ if(NOT Legion_SOURCE_DIR)
   find_package(Legion REQUIRED)
 endif()
 
+find_package(MPI REQUIRED)
+
 add_executable(mpi_interop mpi_interop.cc)
-target_link_libraries(mpi_interop Legion::Legion)
+target_link_libraries(mpi_interop Legion::Legion ${MPI_CXX_LIBRARIES})
+target_include_directories(mpi_interop PRIVATE ${MPI_C_INCLUDE_PATH})
 if(Legion_ENABLE_TESTING)
   add_test(NAME mpi_interop COMMAND $<TARGET_FILE:mpi_interop>)
 endif()
